### ISSUE TITLE
Different outputs for multiple calls

### ISSUE THREAD

**Issue by dulex123** [1ytic/pytorch-edit-distance#2]
When calling compute_wer() multiple times from debugger with the same params I get different results, often nan/infinity and sometimes numbers that greatly vary. I know cuda is not deterministic but this is barely usuable, any ideas what might be causing it? Did I miss some setting before running it?

**Comment by 1ytic** [comment]
Thanks for debugging this package. I use this in another project [open_stt_e2e](https://github.com/1ytic/open_stt_e2e). For example, you can see usage of AverageCER, AverageWER in [train_ctc.py](https://github.com/1ytic/open_stt_e2e/blob/master/train_ctc.py). Hope this helps you.

**Comment by dulex123** [comment]
@1ytic Do you use average because of the instability?

**Comment by 1ytic** [comment]
Nope. AverageCER/WER accumulates error statistics (deletions, insertions, substitutions) for the entire dataset. I recommend you look at the test [test.py](https://github.com/1ytic/pytorch-edit-distance/blob/master/torch_edit_distance/test.py). Maybe it helps you to understand the usage of the package.
